mpl_point_clicker/_clicker.py

```diff
@@ -47,6 +51,10 @@ class in *classes*
             Line2D objects (from ax.plot) are used to generate the markers.
             line_kwargs will be passed through to all of the `ax.plot` calls.
         """
+        if classes is None:
+            classes = 1
+            disable_legend = True
```

Comment:
Move this below where we turn `classes` into a list so we can check the length and thus have a more robust check

```suggestion
        if len(classes) == 1) and disable_legend is None:
            disable_legend = True
```